webapp/channels/src/components/common/scrollbars.scss

```diff
@@ -0,0 +1,21 @@
+.simplebar-track {
+    // Prevent clicking or dragging the scrollbar from interacting with elements behind it
+    pointer-events: all;
+}
+
+.simplebar-track .simplebar-scrollbar::before {
+    width: 6px;
+    border-radius: 2px;
+
+    // Defaults to sidebar text BG, but it can be overridden by the Scrollbars component
+    background-color: rgba(var(--scrollbar-color, var(--sidebar-text-rgb)), 0.32);
+}
+
+.simplebar-track .simplebar-scrollbar.simplebar-visible::before {
+    opacity: 1;
+}
+
+[data-simplebar] {
+    // Ensures that scrollbar divs never overflow their flexbox parents
+    min-height: 0;
```

Comment:
This is some weird CSS thing I learned about. If you have a flexbox with `overflow: hidden` or `overflow: auto` and an overflowing child in it, the child defaults to `min-height: auto` which causes the parent flexbox to expand to fit that child. If you set it to `min-height: 0`, the child will actually be cut off by its parent so it can scroll